docs/digital.rst

```diff
@@ -6,6 +6,10 @@ Board-Specific Pins
 The Raspberry Pi Pico RP2040 chip supports up to 30 digital I/O pins,
 however not all boards provide access to all pins.
 
+Pin Notation
+-----------
```

Comment:
The doc generator complains if the `-`s don't match the size of the name, could you extend these accordingly?

Reply:
Done!